client/goal.c

```diff
@@ -403,7 +417,7 @@ TDNFGoal(
 
         if (nAlterType == ALTER_UPGRADE && dwExcludeCount != 0 && ppszExcludes)
         {
-            /* if we had packages to exclude, then we'd have diabled ones too */
+            /* if we had packages to exclude, then we'd have disabled ones too */
```

Comment:
Chose some other word than `disabled`, some language checking tool complains about this.

Reply:
How about "disenabled"?

Reply:
`/* if we had packages to exclude, then we'd have inactive ones too */` should be fine.